dask/dataframe/tests/test_format.py

```diff
@@ -527,3 +527,16 @@ def test_duplicate_columns_repr():
     arr = da.from_array(np.arange(10).reshape(5, 2), chunks=(5, 2))
     frame = dd.from_dask_array(arr, columns=["a", "a"])
     repr(frame)
+
+
+def test_empty_repr():
+    df = pd.DataFrame()
+    ddf = dd.from_pandas(df, npartitions=1)
+    exp = (
+        "Dask DataFrame Structure:\n"
+        "Empty DataFrame\n"
+        "Columns: []\n"
+        "Index: [, ]\n"
```

Comment:
This bit I'm not sure about... I think it's a bit confusing to say we have an empty index when we may not.

Is the partition info in the rear anywhere?

Reply:
the information about divisions / partitioning is still there. here is the result of the original example in #5761 
```
>>> import dask.dataframe as dd
>>> import pandas as pd
>>> data = {'Name': ['Tom', 'nick', 'krish', 'jack'], 'Age': [20, 21, 19, 18]}
>>> df = pd.DataFrame(data)
>>> dask_df = dd.from_pandas(df, npartitions=1)
>>> categoric_df = dask_df.select_dtypes(include="category")
>>> categoric_df
Dask DataFrame Structure:
Empty DataFrame
Columns: []
Index: [0, 3]
Dask Name: getitem, 2 tasks
```

Reply:
Thanks for posting that. It's a bit misleading since we're saying those values are the Index, when they're actually the divisions.

I think my ideal repr would be something like

```
Empty Dask DataFrame Structure
Columns: []
Divisions: [0, 3]
Dask Name: getitem, 2 tasks
```

Can you also check the `_repr_html_()`?